src/lib/modules/debugger/index.ts

```diff
@@ -252,6 +252,21 @@ class TransactionDebugger {
       usage: "debug <txHash>",
     });
 
+    this.embark.registerConsoleCommand({
+      description: __("During a debug, stop the debugging session"),
```

Comment:
How about:

> Stops an active debugging session.

Reply:
I kept the same syntax as other command here:
e.g During a debug, step over forward

Reply:
Understood. I'd even say that those other places would need fine-tuning as well.

Your call :)